src/executor/source/Index.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express'
+import MorganMiddleware from './utils/logging/MorganMiddleware'
+import version from './routes/VersionRouter'
+import Logger from './utils/logging/Logger'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+const app = express()
+app.disable( 'x-powered-by' )
+
+app.use( MorganMiddleware )
+
+app.get( '/status', ( req: any, res: any ): void => {
+  res.send( 'running' )
+} )
+
+app.use( '/version', version )
```

Comment:
Why do we use `app.use` instead of `app.get` here? Do we expect other request types or sth?